templates/cluster-template-managed-private.yaml

```diff
@@ -79,12 +79,11 @@ metadata:
   namespace: default
 spec:
   version: "${KUBERNETES_VERSION}"
-  nodeShape: "${OCI_MANAGED_NODE_SHAPE}"
+  nodeShape: "${OCI_MANAGED_NODE_SHAPE=VM.Standard.E4.Flex}"
   sshPublicKey: "${OCI_SSH_KEY}"
   nodeSourceViaImage:
     imageId: "${OCI_MANAGED_NODE_IMAGE_ID}"
     bootVolumeSizeInGBs: ${OCI_MANAGED_NODE_BOOT_VOLUME_SIZE=50}
   nodeShapeConfig:
-    memoryInGBs: "${OCI_MANAGED_NODE_MACHINE_MEMORY=16}"
```

Comment:
Should this be removed?

Reply:
yeah, it makes it similar to other templates which makes the memory defaulted based on ocpu right?